src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -456,6 +473,21 @@ else if (isInSameGroup(groupIdx, isStatic)) {
         lastImport = name;
     }
 
+    /**
+     * Checks whether import groups should be separated.
+     * @param isStatic whether the token is static or not.
+     * @return true if imports groups should be separated.
+     */
+    private boolean checkSeparatedBetweenGroups(boolean isStatic) {
```

Comment:
Maybe it will be better to rename to "shouldBeSeperatedBetweenGroups" or "shouldSeparationPresentBetwenGrooups", "shouldSeparationBeBetweenGrooups", etc. Method names which return boolean usually start with "is", "should", "cant", "exist", etc.

Reply:
done and thank you.